src/xdocs/config_metrics.xml

```diff
@@ -101,6 +101,22 @@
         <source>
 &lt;module name=&quot;BooleanExpressionComplexity&quot;/&gt;
         </source>
+        <p>Code Example:</p>
+        <div class="wrapper">
+          <pre>
+            public class Test
```

Comment:
All examples should start at the first column and not start off this far indented. Look at the screenshot and how the example is so far to the right.

Reply:
Current Look:
![Screenshot_2020-03-03 checkstyle – Metrics(1)](https://user-images.githubusercontent.com/45921807/75730864-17e78780-5d14-11ea-84e5-daa9274bc5c2.png)
![Screenshot_2020-03-03 checkstyle – Metrics(2)](https://user-images.githubusercontent.com/45921807/75730865-18801e00-5d14-11ea-97fb-27fea98123a0.png)
![Screenshot_2020-03-03 checkstyle – Metrics(3)](https://user-images.githubusercontent.com/45921807/75730867-1918b480-5d14-11ea-92cd-3e688ed7237f.png)

Reply:
@pietern please update your initial post with the current screenshot .